test/demo/utils/delay.ts

```diff
@@ -13,6 +13,7 @@ export enum Duration {
     PAGE_NAVIGATE = 500,
     SHORTER_THAN_APP_MATURITY = 2000,
     LONGER_THAN_APP_MATURITY = 6000,
+    LONGER_THAN_SERVICE_TO_CLIENT_API_CALL_TIMEOUT = 6000,
```

Comment:
🤣

Reply:
Can we just combine some of these into a single "long delay"

Reply:
As is, this communicates something about why we're delaying, which a single "long delay" wouldn't. Would prefer to leave as is

Reply:
Hmmm. I just seems unnecessarily verbose, but not worth arguing over.